libtorrent-rasterbarConfig.cmake

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/libtorrent-rasterbarTargets.cmake")
```

Comment:
It's generated by cmake when you do a "make install" or equivalent. You can see the reference to it on line 382 in CMakeLists.txt